### PR TITLE
New route card styling and some additional features

### PR DESCRIPTION
**Pull request by Wouter125**
Currently migrating the examples to their new UI. Started with the routing one. Also embedded a few new additional features that were in the design:

- [x] Average temperature across all stops calculation with our own data 
- [x] Detect whether we have a border crossing with our own data
- [x] Update the map markers to reflect the charging time at all times. Already took out the on hover.
- [x] Moved route example from OCM to EcoMovement data.

### CODE REVIEW

File: public/route/style.css [hunk 1]
```diff
@@ -2,55 +2,100 @@ html,
...
 }
 
-h2 {
-  padding: 8px 16px 0;
-  margin-bottom: 10px;
-  font-size: 18px;
+h3 {
   font-weight: 600;
-  line-height: 24px;
+  font-size: 1.25rem;
```
Comment (lsmienk) [inline public/route/style.css:36]: We are using px everywhere else. Let's stick to that.

File: route/index.js [hunk 2]
```diff
@@ -116,34 +116,90 @@ const drawRoutePolyline = data => {
...
+  const routeEnergy = data.consumption ? `${data.consumption.toFixed(2)} kWh` : 'Unknown';
 
-  // the amount of stops in this route
-  document.getElementById('stop').innerHTML = `${data.charges ?? 0} stops`;
+  // the average temperature based on all stops across the route
+  const averageTemperature = calculateAverageTemperature(data);
+
+  const hasBorderCrossing = checkBorderCrossing(data);
+
+  console.log(hasBorderCrossing);
```
Comment (lsmienk) [inline route/index.js:133]: clean up

File: route/index.js [hunk 3]
```diff
@@ -116,34 +116,90 @@ const drawRoutePolyline = data => {
...
+};
+
+/**
+ * Calculate the average temperature by adding up all temperatures across every stop
+ * and dividing them by the number of stations.
+ * @param { object } data - route specification
+ * @returns { int } - average temperature rounded across all stops to 1 decimal
+ */
+const calculateAverageTemperature = data => {
+  var averageTemperature = 0;
```
Comment (lsmienk) [inline route/index.js:177]: Please use cons or let

Comment (Wouter125) [reply route/index.js:177]: Check will turn it into a let.

File: public/route/style.css (hunk 1 above)
Comment (Wouter125) [reply public/route/style.css:36]: Hm, I switched like everything to REM / EM. We doing the same on the website actually. The reason I did this is whenever we adjust our base-font or spacing will be relative without having to re-do the whole UI again.

Was about to do something similar to the other examples. Planning on moving the colors to the `:root` as well. If we really prefer `px` I'll revert the changes.

File: route/index.js (hunk 2 above)
Comment (Wouter125) [reply route/index.js:133]: Check, will do.

**Review by lsmienk — COMMENTED** [review]
I left a few comments and will look at it further when the pr is finished :)

File: public/route/style.css (hunk 1 above)
Comment (lsmienk) [reply public/route/style.css:36]: I think rem is okay too. But then indeed we should be using it everywhere.

**Comment by Wouter125** [comment]
> I left a few comments and will look at it further when the pr is finished :)

Think we got the new design in place now. Let me know if we missing anything. Otherwise I'll move onto the next example.

**Review by lsmienk — APPROVED** [review]
The changes that are made here (the different route for example) can also be added to the readme for this example.

**Comment by Wouter125** [comment]
Readme has been updated and we included a link to github. Think it's ready for merging.

**Pull request merged**